website/docs/language/settings/backends/azurerm.mdx

```diff
@@ -11,229 +11,498 @@ This backend supports state locking and consistency checking with Azure Blob Sto
 
 ~> **Terraform 1.1 and 1.2 supported a feature-flag to allow enabling/disabling the use of Microsoft Graph (and MSAL) rather than Azure Active Directory Graph (and ADAL) - however this flag has since been removed in Terraform 1.3. Microsoft Graph (and MSAL) are now enabled by default and Azure Active Directory Graph (and ADAL) can no longer be used.
 
-## Example Configuration
+## Authentication
 
-When authenticating using the Azure CLI or a Service Principal (either with a Client Certificate or a Client Secret):
+The `azurerm` backend supports 3 methods of authenticating to the storage account:
+
+- **Access Key** (default)
+- **Entra ID**
+- **SAS Token**
+
+The **Access Key** method can be used directly or in combination with an Entra ID principal (e.g. user, service principal or managed identity). By default when you use an Entra ID principal, it will use the Access Token of that principal to generate an Access Key for the storage account and use that Access Key to authenticate to the state file blob. To use an Access Key directly you must generate one for you state file blob and pass it to the backend config.
```

Comment:
```suggestion
The **Access Key** method can be used directly or in combination with an Entra ID principal (e.g. user, service principal or managed identity). By default when you use an Entra ID principal, it will use the Access Token of that principal to generate an Access Key for the storage account and use that Access Key to authenticate to the state file blob. To use an Access Key directly you must generate one for your state file blob and pass it to the backend config.
```